seagoat/sources/chroma.py

```diff
@@ -1,19 +1,21 @@
 from pathlib import Path
+from typing import List
 
 import chromadb
 from chromadb.config import Settings
 from chromadb.errors import IDAlreadyExistsError
+from llama_cpp import Llama
 
 from seagoat.cache import Cache
 from seagoat.repository import Repository
 from seagoat.result import Result
 
-
 MAXIMUM_VECTOR_DISTANCE = 1.5
 
 
 def initialize(repository: Repository):
     cache = Cache("chroma", Path(repository.path), {})
+    embedding_function = get_embedding_function(repository)
```

Comment:
`get_config_values` can be used for this, it just needs the repo path, which is already provided. But maybe like a "config object" or sth like that could be created so that we can pass more data to data sources. Anyways for now I think it's ok to add a little bit of redudancy and get the config here

Reply:
Also I would look at this comment as it seems to be relevant, looks like there are people interested in configuring the model, and maybe it' relevant to also allow users to customize the embedding function in general, like even customizing whether to use GPU and such :thinking: not a requirement for merging necessarily